AppDashboard/dashboard.py

```diff
@@ -121,6 +123,7 @@ def render_template(self, template_file, values=None):
       'is_user_cloud_admin': self.dstore.is_user_cloud_admin(),
       'can_upload_apps': self.dstore.can_upload_apps(),
       'apps_user_is_admin_on': owned_apps,
+      'user_layout_pref': json.loads(self.dstore.get_dash_layout_settings(), object_pairs_hook=OrderedDict),
```

Comment:
Please limit all code lines to 80 characters.
In this case you could modify the line to be:

```
'user_layout_pref': json.loads(self.dstore.get_dash_layout_settings(), 
                               object_pairs_hook=OrderedDict),
```